ui/src/views/compute/DeployVM.vue

```diff
@@ -562,10 +562,97 @@
                           @change="val => { dynamicscalingenabled = val }"/>
                       </a-form-item>
                     </a-form-item>
-                    <a-form-item :label="$t('label.userdata')" name="userdata" ref="userdata">
-                      <a-textarea
-                        v-model:value="form.userdata">
-                      </a-textarea>
+                    <a-form-item :label="$t('label.userdata')">
```

Comment:
this works well.

it looks the form to update vm,  register Iso, upload iso from local, update iso are not changed.

Reply:
again, updateVM changes are lost during rebase. I'll add them again.